testsuite/features/support/twopence_init.rb

```diff
@@ -189,6 +189,8 @@ def file_inject(node, local_file, remote_file)
 $private_net = ENV['PRIVATENET'] if ENV['PRIVATENET']
 $mirror = ENV['MIRROR']
 $server_http_proxy = ENV['SERVER_HTTP_PROXY'] if ENV['SERVER_HTTP_PROXY']
+$noauth_registry = ENV['REGISTRY_URI'] if ENV['REGISTRY_URI']
+$auth_registry = ENV['PORTUS_URI'] if ENV['PORTUS_URI']
```

Comment:
We could consider renaming `PORTUS_URI` and `REGISTER_URI` variables accordingly (to `AUTH_REGISTERY` and `NOAUTH_REGISTERY`), but that would mean changes in sumaform

Reply:
Done